File: Web/src/main/java/com/itechart/webflux/web/core/repository/MeetingUserRepository.java
package com.itechart.webflux.web.core.repository;

import com.itechart.webflux.web.core.model.MeetingUser;
import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
import reactor.core.publisher.Flux;

public interface MeetingUserRepository extends ReactiveMongoRepository<MeetingUser, String> {

    Flux<MeetingUser> findByUserId(String userId);

    void deleteByUserId(String userId);
}

File: Web/src/main/java/com/itechart/webflux/web/core/repository/MeetingUserRepository.kt
package com.itechart.webflux.web.core.repository

import com.itechart.webflux.web.core.model.entity.MeetingUser
import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository
import reactor.core.publisher.Flux

@Repository
interface MeetingUserRepository : ReactiveMongoRepository<MeetingUser, String> {

    fun findByUserId(userId: String): Flux<MeetingUser>

    fun deleteByUserId(userId: String)

}